### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by kalyaneswar**

### CODE REVIEW

**Comment by Kalyaneswar1** [comment]
Why code is changed now?

**Comment by kalyaneswar** [comment]
Bcz to add more varities in module.

**Comment by kalyaneswar** [comment]
> Why code is changed now?

just to add extra modules.

File: menu.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-dosa
\ No newline at end of file
+dosa
+karam
+oil
```
Comment (kalyaneswar) [inline menu.txt:3]: why oil again in the end?

Comment (Kalyaneswar1) [inline menu.txt:3]: why oil again added in the end if present already

**Review by Kalyaneswar1 — APPROVED** [review]
all looks good, proceed with the changes

**Pull request merged**